packages/gatsby-transformer-asciidoc/src/gatsby-node.js

```diff
@@ -42,7 +42,7 @@ async function onCreateNode(
   // Load Asciidoc file for extracting
   // https://asciidoctor-docs.netlify.com/asciidoctor.js/processor/extract-api/
   // We use a `let` here as a warning: some operations, like .convert() mutate the document
-  let doc = await asciidoc.load(content, asciidocOptions)
+  let doc = await asciidoc.load(content, _.assign({base_dir: node.dir}, asciidocOptions))
```

Comment:
We can use spread operator here I think (no need to use `_.assign`) - let's use js language features if possible over utils from packages :
```suggestion
  let doc = await asciidoc.load(content, {
    base_dir: node.dir,
    ...asciidocOptions
  })
```